### PR TITLE
Create signal/index.hbs

### PR DESCRIPTION
**Pull request by natewcl**
@julieritchie @kellyfinn please review Signal entry

### CODE REVIEW

**Comment by natewcl** [comment]
@julieritchie @kellyfinn Also adding ComScore PD entry for review. Logos send via email. Thanks!

File: website/partners/technology/signal/index.hbs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: partners.hbs
+title: "Signal"
+integrations: true
+priority: 60.5
+summary: "Pass data to partners in real-time via Signal's TMS, reducing the number of tags on site and stopping data loss."
+logo: ""
+website_link: "https://www.signal.co/"
+website_display: "https://www.signal.co/"
+kb_article: ""
```
Comment (dtothefp) [inline website/partners/technology/signal/index.hbs]: @natewcl please make sure `website_display` does not have `https://` so it will only display `www`

**Comment by dtothefp** [comment]
@natewcl is this ready for content review?

**Comment by kellyfinn** [comment]
This is ready @dtothefp @stewsmith @julieritchie

**Comment by dtothefp** [comment]
retest this please

**Comment by dtothefp** [comment]
@kellyfinn the page was broken I'm thinking because there was a line break in the yml at the quotes but I'm rebuilding to check

screen shot 2015-04-01 at 4 45 20 pm

**Comment by dtothefp** [comment]
@kellyfinn please remember to remove HTTP and HTTPS protocols for links. I fixed this and am merging

**Pull request merged**